config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  get '/events', to: 'events#index'
+  get '/', to: 'events#index'
+  get '/show', to: 'events#show'
```

Comment:
show method should return the detail of an object, check the official documentation for this modification please
[Rails Documentation](https://guides.rubyonrails.org/routing.html#connecting-urls-to-code)